Docker/Alfresco/acs-repository/ark-startup.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+# Configue Repository to listen on port 8081 instead of 8080
+sed -i 's/port="8080"/port="8081"/g' /usr/local/tomcat/conf/server.xml
+
+# Remove AJP port
+sed -i 's|^.*Connector.*port="8009".*$||' /usr/local/tomcat/conf/server.xml
+
+# Remove shutdown port
+sed -i 's/port="8005"/port="-1"/g' /usr/local/tomcat/conf/server.xml
+
+[ -v JAVA_OPTS ] || JAVA_OPTS=
+
+if [ -v ARK_DB_SECRET_ARN ]; then
+    echo "I am being run on ECS"
+
+    echo "Fetching the DB credentials and adding them to JAVA_OPTS"
+    response=$(aws secretsmanager get-secret-value --secret-id "$ARK_DB_SECRET_ARN")
+    secret=$(echo "$response" | jq -r .SecretString)
+    host=$(echo "$secret" | jq -r .host)
+    port=$(echo "$secret" | jq -r .port)
+    dbname=$(echo "$secret" | jq -r .dbname)
+    username=$(echo "$secret" | jq -r .username)
+    password=$(echo "$secret" | jq -r .password)
+    JAVA_OPTS="$JAVA_OPTS -Ddb.url=\"jdbc:mariadb://$host:$port/$dbname?autoReconnect=true&useUnicode=true&characterEncoding=UTF-8&useSSL=true&requireSsl=true&enabledSslProtocolSuites=TLSv1.2&trustServerCertificate=false&serverSslCert=/etc/pki/ca-trust/source/anchors/rds-ca-2019-root.pem\""
+    JAVA_OPTS="$JAVA_OPTS -Ddb.username='$username' -Ddb.password='$password'"
+    echo "MariaDB host: $host"
+    echo "MariaDB port: $port"
+    echo "MariaDB dbname: $dbname"
+    echo "MariaDB username: $username"
+    echo 'Added `-Ddb.url`, `-Ddb.username` and `-Ddb.password` to `JAVA_OPTS`'
+else
+    echo "I am being run locally using docker-compose"
```

Comment:
Eventually we will support other container orchestrators in addition to ECS and Docker Compose but I guess we will change this script then!